shell/renderer/api/electron_api_context_bridge.h

```diff
@@ -16,21 +16,12 @@ namespace electron {
 
 namespace api {
 
-namespace context_bridge {
-class RenderFrameFunctionStore;
-}
-
-v8::Local<v8::Value> ProxyFunctionWrapper(
-    context_bridge::RenderFrameFunctionStore* store,
-    size_t func_id,
-    bool support_dynamic_properties,
-    gin_helper::Arguments* args);
+void ProxyFunctionWrapper(const v8::FunctionCallbackInfo<v8::Value>& info);
 
 v8::MaybeLocal<v8::Object> CreateProxyForAPI(
```

Comment:
These functions are never referenced outside of the .cc file either. Can we just delete this header?

Reply:
@nornagon These functions reference each other so need to be "forward declared" in a sense

Reply:
It's totally fine to forward-declare inside the C++ file—the header is only necessary if you're referencing the functions from somewhere else.